tests/speculos/test_configuration_cmd.py

```diff
@@ -1,3 +1,3 @@
 
 def test_configuration(cmd):
-    assert cmd.get_configuration() == (2, 1, 11, 0)
+    assert cmd.get_configuration() == (2, 1, 11, 1)
```

Comment:
You can use something like here https://github.com/LedgerHQ/app-exchange/blob/41d158b4e7eb54f78c2ef5f6f4a1c93819a64f63/test/python/utils.py#L29

Reply:
Of course, and this is already planned in another Jira ticket B2CA-901, related to the ragger tests.